packages/ui/src/components/Command/Command.utils.tsx

```diff
@@ -69,6 +69,7 @@ export const CommandDialog = ({ children, onKeyDown, page, ...props }: CommandDi
         '!border-overlay/90',
         'transition ease-out',
         'place-self-start mx-auto top-24',
+        'max-w-[80%] xs:max-w-[50%]',
```

Comment:
Can we have the max-width be same width as the content below? 
So something like `max-w-[calc(100vw-40px)]` (if that's the correct padding, need to double check)

Reply:
@fsansalvadore - Just pushed a commit for the above, upped from `40px` to `60px`

Reply:
could do something like a `max-w-[min(100vw-40px,600px)]`?

sorry if that's dodgy tailwind, wrote it off the top of my head